### PR TITLE
Update realms.mdx

### PR DESCRIPTION
**Pull request by waveform06**
Some additional clarifications

### CODE REVIEW

File: docs/vote-escrow/realms.mdx [hunk 1]
```diff
@@ -155,7 +159,7 @@ will be added.
 
 ### Stake Multiplier
 
-veHNT positions can be strengthened by increasing the duration that thet are locked for. The
+veHNT positions can be strengthened by increasing the duration that they are locked for. The
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx:162]: 🙃

File: docs/vote-escrow/realms.mdx [hunk 2]
```diff
@@ -250,6 +267,11 @@ constant lockup position will first need to be unlocked before the cooldown peri
 Unlocking the position will begin the unlocking process. The time this takes will depend on the
 duration set during the creation of the position.
 
+You must undelegate before extending a stake. You can redelegate after the extend is complete.
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx]:
```suggestion
```
It is confusing that we're talking about extending stakes in this section.

File: docs/vote-escrow/realms.mdx [hunk 3]
```diff
@@ -250,6 +267,11 @@ constant lockup position will first need to be unlocked before the cooldown peri
...
+
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx]:
```suggestion
```

File: docs/vote-escrow/realms.mdx [hunk 4]
```diff
@@ -240,6 +253,10 @@ positions exist in an account, the "Transfer" button will be disabled for a posi
 As expected, extending a stake lengthens the amount of time it will take to unstake. Position
 holders may chose to extend a stake in order to gain a larger multiplier for voting power or
 delegation purposes.
+When you select the extend stake period you are updating the period of lockup of the stake,
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx]: This sentence is confusing, can we reword?

File: docs/vote-escrow/realms.mdx [hunk 5]
```diff
@@ -218,15 +222,24 @@ veHNT position to multiple networks (IOT or MOBILE) or set different lockup time
...
-Stakes can be transfered from one position into another within the same account. Positions can only
-be transfered to positions with a greater or equal duration. Only undelegated positions can be
-transfered. Staked tokens cannot be moved to other accounts. Landrush positions cannot be
-transfered. This functionality is useful for consolidating positions. If no other compatible
-positions exist in an account, the "Transfer" button will be disabled for a position.
+Staked tokens cannot be moved to other wallets, neither can the NFT representing a stake be transfered 
+to another wallet. Stakes can only be transfered from one position into another within the same account. 
+Positions can only be transfered to positions with a greater or equal duration. This functionality is 
+useful for consolidating positions. 
+Landrush positions cannot be transfered into after the landrush period. Any veHNT transfered out of a
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx]: Reword more clearly, e.g. "More tokens cannot be transferred into a land rush position..."

**Review by samgutentag — APPROVED** [review]

**Pull request merged**